test/method/testvolume.py

```diff
@@ -104,3 +104,8 @@ def test_roundtrip_cube(self):
         density_recovered = volume.read_from_cube("coarsewater.cube")
 
         assert_allclose(density.data, density_recovered.data, rtol=0.05)
+
+    def test_zip_cube(self):
+        """Check we can read from a zipped file."""
+        data = volume.read_from_cube(f"{os.path.dirname(os.path.realpath(__file__))}/co.cube.zip")
```

Comment:
We should use `pathlib.Path` in as many places as possible:
```suggestion
        data = volume.read_from_cube(Path(__file__).resolve().parent / "co.cube.zip")
```

Reply:
I'm a pathlib fanboy so no complaints from me